docs/devs/e2e-tests.md

```diff
@@ -0,0 +1,58 @@
+# E2E Tests
+
+## Introduction
+    E2E testing is a critical aspect of Ozone HIS that ensures all the workflows work as expected.
+
+## Prerequisites
+    Before running Ozone E2E tests, ensure you have Node.js and npm installed on your machine.
```

Comment:
It would be helpful to specify the minimum required versions of `Node.js` and `npm`.